src/uia.cpp

```diff
@@ -241,11 +239,10 @@ bool sendUiaNotification(const string& message, bool interrupt) {
 	if (!UiaClientsAreListening() || message.empty()) {
 		return true;
 	}
-	return (uiaCore->RaiseNotificationEvent(
-		uiaProvider,
-		NotificationKind_Other,
-		interrupt ? NotificationProcessing_MostRecent : NotificationProcessing_All,
-		SysAllocString(widen(message).c_str()),
-		SysAllocString(L"REAPER_OSARA")
-	) == S_OK);
+	return (
+		uiaCore->RaiseNotificationEvent(uiaProvider, NotificationKind_Other,
+			interrupt ? NotificationProcessing_MostRecent
+								: NotificationProcessing_All,
```

Comment:
I wish I knew how we could avoid these random tabs being added here. I'd say AlignOperands would do the trick nicely here, but it seems it doesn't.

Reply:
So AlignOperands: DontAlign doesn't work?

Reply:
We definitely shouldn't touch ContinuationIndentWidth, that makes things worse.